collections/_components/button.html

```diff
@@ -50,25 +50,22 @@
 {% endcapture %}
 
 {% capture button-variants %}
-<button type="button" class="btn btn-primary sgds">primary</button>
-<button type="button" class="btn btn-secondary sgds">secondary</button>
-<button type="button" class="btn btn-success sgds">success</button>
-<button type="button" class="btn btn-danger sgds">danger</button>
-<button type="button" class="btn btn-warning sgds">warning</button>
-<button type="button" class="btn btn-info sgds">info</button>
-<button type="button" class="btn btn-light sgds">light</button>
-<button type="button" class="btn btn-dark sgds">dark</button>
-{% endcapture %}
-
-{% capture button-outline %}
-<button type="button" class="btn btn-outline-primary sgds">Primary</button>
-<button type="button" class="btn btn-outline-secondary sgds">Secondary</button>
-<button type="button" class="btn btn-outline-success sgds">Success</button>
-<button type="button" class="btn btn-outline-warning sgds">Warning</button>
-<button type="button" class="btn btn-outline-danger sgds">Danger</button>
-<button type="button" class="btn btn-outline-info sgds">Info</button>
-<button type="button" class="btn btn-outline-light sgds">Light</button>
-<button type="button" class="btn btn-outline-dark sgds">Dark</button>
+<button type="button" class="btn btn-primary sgds">Primary</button>
+<button type="button" class="btn btn-secondary sgds">Secondary</button>
+<button type="button" class="btn btn-success sgds">Success</button>
+<button type="button" class="btn btn-danger sgds">Danger</button>
+<button type="button" class="btn btn-warning sgds">Warning</button>
+<button type="button" class="btn btn-info sgds">Info</button>
+<button type="button" class="btn btn-light sgds">Light</button>
+<button type="button" class="btn btn-dark sgds">Dark</button>
+<button type="button" class="btn btn-outline-primary sgds">Outline-primary</button>
+<button type="button" class="btn btn-outline-secondary sgds">Outline-secondary</button>
+<button type="button" class="btn btn-outline-success sgds">Outline-success</button>
+<button type="button" class="btn btn-outline-warning sgds">Outline-warning</button>
+<button type="button" class="btn btn-outline-danger sgds">Outline-danger</button>
+<button type="button" class="btn btn-outline-info sgds">Outline-info</button>
+<button type="button" class="btn btn-outline-light sgds">Outline-light</button>
```

Comment:
Could you help add this liner to the variant description?
"Some of the outline buttons like <code>btn-outline-light</code> should be used in a darker background for sufficient color contrast"

Reply:
added in the liner